compiler/noirc_frontend/src/elaborator/comptime.rs

```diff
@@ -30,6 +30,24 @@ use crate::{
 
 use super::{Elaborator, FunctionContext, ResolverMeta};
 
+#[derive(Debug, Copy, Clone)]
+pub(crate) struct AttributeContext {
```

Comment:
```suggestion
struct AttributeContext {
```
Looks like this isn't used outside this module